Makefile

```diff
@@ -239,50 +173,186 @@ DOCKER_BUILD_ARGS ?= --platform=linux/amd64
 ifneq ($(CLUSTER_TYPE),)
 	DOCKER_BUILD_ARGS = --platform=$(CLUSTER_OS)/$(CLUSTER_ARCH)
 endif
+.PHONY: docker-build
 docker-build: ## Build docker image with the manager.
 	$(CONTAINER_TOOL) build --load -t $(IMG) . $(DOCKER_BUILD_ARGS)
 
+.PHONY: docker-push
 docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
+# PLATFORMS defines the target platforms for the manager image be built to provide support to multiple
+# architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
+# - be able to use docker buildx. More info: https://docs.docker.com/build/buildx/
+# - have enabled BuildKit. More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# - be able to push the image to your registry (i.e. if you do not set a valid value via IMG=<myregistry/image:<tag>> then the export will fail)
+# To adequately provide solutions that are compatible with multiple platforms, you should consider using this option.
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
+docker-buildx: ## Build and push docker image for the manager for cross-platform support
```

Comment:
Bump instructions to 1.34.0 mensions replacing this:

https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.34.0/

Reply:
I think we can safely ignore this

we do not use this command (CI builds multi arch images for us and we use `make docker-build` and `make bundle-build` when we want different arch images)

we could remove the command. I prefer to let it be so future Operator SDK upgrades are easier

Reply:
I think it's still worth updating it to say we are fully migrated to 1.34.x.
One could use the buildx to work on multi-arch, so either remove that or modify to ensure it's aligned with the upgrading release instructions and then I will approve.

Thanks.

Reply:
removed